firmware/targets/furi_hal_include/furi_hal_rtc.h

```diff
@@ -12,6 +12,17 @@
 extern "C" {
 #endif
 
+#define FURI_HAL_RTC_SECONDS_PER_MINUTE 60
+#define FURI_HAL_RTC_SECONDS_PER_HOUR (FURI_HAL_RTC_SECONDS_PER_MINUTE * 60)
+#define FURI_HAL_RTC_SECONDS_PER_DAY (FURI_HAL_RTC_SECONDS_PER_HOUR * 24)
+#define FURI_HAL_RTC_EPOCH_START_YEAR 1970
+#define FURI_HAL_RTC_IS_LEAP_YEAR(year) \
+    ((((year) % 4 == 0) && ((year) % 100 != 0)) || ((year) % 400 == 0))
+
+#define FURI_HAL_RTC_MONTHS_COUNT 12
+
+extern const uint8_t furi_hal_rtc_days_per_month[2][FURI_HAL_RTC_MONTHS_COUNT];
+
```

Comment:
I wouldn't move it here.
Better to add some functions in API:
bool furi_hal_rtc_is_leap_year();
uint8_t furi_hal_rtc_get_days_per_months()
@skotopes  what do you think?

Reply:
Done